### PR TITLE
UIRR-8 - Implement "Allow" and "Deny" actions for saving access log.

### PR DESCRIPTION
**Pull request by Terala-Priyanka**
## Purpose
[UIRR-7](https://folio-org.atlassian.net/browse/UIRR-8) - Implement Allow and Deny Actions.

## Screencast
lK5xJybKKP

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by manvendra-s-rathore — APPROVED** [review]

**Comment by Terala-Priyanka** [comment]
Hi @folio-org/fe-tl-reviewers 
Could you please review this PR?

File: src/Footer/Footer.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import React, { useCallback } from 'react';
```
Comment (artem-blazhko) [inline src/Footer/Footer.js:1]: Could you remove `React`?

File: src/Footer/Footer.js [hunk 2]
```diff
@@ -8,8 +9,28 @@ import {
 } from '@folio/stripes/components';
 
 import css from './Footer.css';
+import { ALLOWED, DENIED } from '../../constants';
```
Comment (artem-blazhko) [inline src/Footer/Footer.js:12]: Lets follow general import order.
Would be great to import css file at the end of all imports.

File: src/Footer/Footer.js [hunk 3]
```diff
@@ -29,7 +50,10 @@ const Footer = ({ resetDetails, form }) => {
             id="deny-access"
             type="submit"
             buttonStyle="danger mega"
-            onClick={() => {}}
+            onClick={(e) => {
+              e.preventDefault();
+              handleAccessLog(DENIED);
+            }}
```
Comment (artem-blazhko) [inline src/Footer/Footer.js:56]: Could you create a separate function out of `return` statement as we discussed before?

File: src/Footer/Footer.js [hunk 4]
```diff
@@ -39,7 +63,11 @@ const Footer = ({ resetDetails, form }) => {
             id="allow-access"
             type="submit"
             buttonStyle="primary mega"
-            onClick={() => {}}
+            onClick={(e) => {
+              e.preventDefault();
+              handleAccessLog(ALLOWED);
+            }}
```
Comment (artem-blazhko) [inline src/Footer/Footer.js:69]: Could you create a separate function out of `return` statement as we discussed before?

**Review by artem-blazhko — APPROVED** [review]

**Pull request merged**